### PR TITLE
Added Nginx pillar for edx video app to enable TLS

### PR DESCRIPTION
**Pull request by blarghmatey**
The edX Ansible code doesn't have a way to configure TLS for the video web ui so I added it via Salt

### CODE REVIEW

File: pillar/nginx/edx_veda.sls [hunk 1]
```diff
@@ -0,0 +1,109 @@
+{% set app_name = 'mitx-cas' %}
```
Comment (shaidar) [inline pillar/nginx/edx_veda.sls]: Copy pasta?

File: pillar/nginx/edx_veda.sls [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+{% set env_settings = salt.cp.get_file_str("salt://environment_settings.yml")|load_yaml %}
+{% set ENVIRONMENT = salt.grains.get('environment', 'mitx-qa') %}
+{% set env_data = env_settings.environments[ENVIRONMENT] %}
+{% set server_domain_names = env_data.purposes['mitx-cas'].domains %}
```
Comment (shaidar) [inline pillar/nginx/edx_veda.sls]: Copy pasta?

File: pillar/nginx/edx_veda.sls [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+{% set server_domains = {
+  'mitx-qa': server_domain_names[0],
+  'mitx-production': server_domain_names[1]
```
Comment (shaidar) [inline pillar/nginx/edx_veda.sls]: Shouldn't those be `mitxpro-`?

File: salt/environment_settings.yml [hunk 4]
```diff
@@ -136,8 +136,11 @@ environments:
         bucket_uses:
           - grades
           - storage
-          - video-upload
-          - video-delivery
+          - edx-video-upload
+          - edx-video-delivery
```
Comment (shaidar) [inline salt/environment_settings.yml]: What's the use of the new edx-video-*?

Comment (blarghmatey) [reply salt/environment_settings.yml]: Those are for the destinations for edx to upload and play video going to/from the video pipeline.

**Comment by shaidar** [comment]
👍

**Pull request merged**